src/games/strategy/triplea/delegate/StrategicBombingRaidBattle.java

```diff
@@ -327,23 +336,32 @@ private void showBattle(final IDelegateBridge bridge) {
     DiceRoll m_dice;
     CasualtyDetails m_casualties;
     Collection<Unit> m_casualtiesSoFar = new ArrayList<>();
+    Collection<Unit> validAttackingUnitsForThisRoll;
```

Comment:
please mark this as final. The auto-clean up tool linked in the docs would apply that for you..